examples/todos-app/src/styles/tailwind-output.css

```diff
@@ -433,11 +433,6 @@ input::-moz-placeholder, textarea::-moz-placeholder {
   color: #9ca3af;
 }
 
-input:-ms-input-placeholder, textarea:-ms-input-placeholder {
```

Comment:
I have noticed that diff are being generated so I am pushing the fixes together.